File: examples/documents/list-group-documents.ts
import type { JsBaoClient } from "js-bao-wss-client";

// Documents the user can access via a group they belong to.
export async function listGroupDocuments(client: JsBaoClient) {
  // #region example
  const documents = await client.groups.listDocuments("team", "engineering");
  // #endregion example
  return documents;
}

File: examples/documents/list-group-documents.swift
import JsBaoClient

// Documents the user can access via a group they belong to.
func listGroupDocuments(client: JsBaoClient) async throws {
  // #region example
  let documents = try await client.groups.listDocuments(groupType: "team", groupId: "engineering")
  // #endregion example
  _ = documents
}
